tests/test_launcher.py

```diff
@@ -19,6 +19,10 @@ def test_manual_fluent_version_setting():
     with pytest.raises(RuntimeError):
         pyfluent.set_ansys_version(22.1)
 
+    # Resets the global variable to its original state
+    pyfluent.set_ansys_version(version=pyfluent.FluentVersion.version_22R2)
```

Comment:
What if we are originally running Fluent 23.1?

Reply:
ohk. I'll update it accordingly.
Thank you